### DISCUSSION TITLE
Fix black screen error in game ben 10 on snapdragon 660 please 🙏🏻

### DISCUSSION THREAD

**Discussion by 20008gui** [skyline-emu/skyline#1575] [Q&A]
ID 01006E1004404000

The game is already running on some mobile phones and it is semi playable or even playable.

Some Of The Phones Are:
Xiaomi Poco X3 Pro
Samsung  F23
And Asus Rog Phone 5

And I want to run this game on xiaomi redmi note 7 please 🙏🏻

**Comment by Allatel** [comment]
sd 660 is a no-no

**Comment by Helloooo908** [comment]
> 

Brother, that mobile has a not so powerful processor, but if you are on android 10 maybe it will run in the future when they add the adrenotools interface to use old gpu drivers, but I don't think it will run heavy 3D games.

**Comment by Crytonics** [comment]
Redmi note 7 have old Adreno with old drivers. 3D games won't work or will some but with low fps. So if you want to play 3D games you need better phone.

**Reply by PlayGame666** [reply]
Will Snapdragon 732G work?  Android version 12, Adreno (TM) 618 and 8 GB RAM is enough?

**Reply by hope3x** [reply]
> Will Snapdragon 732G work?  Android version 12, Adreno (TM) 618 and 8 GB RAM is enough?

It should work.
